src/main/java/net/fabricmc/loom/api/decompilers/DecompilationMetadata.java

```diff
@@ -27,14 +27,6 @@
 import java.nio.file.Path;
 import java.util.Collection;
 
-public class DecompilationMetadata {
-	public final int numberOfThreads;
-	public final Path javaDocs;
-	public final Collection<Path> libraries;
-
-	public DecompilationMetadata(int numberOfThreads, Path javaDocs, Collection<Path> libraries) {
-		this.numberOfThreads = numberOfThreads;
-		this.javaDocs = javaDocs;
-		this.libraries = libraries;
-	}
+public record DecompilationMetadata(int numberOfThreads, Path javaDocs,
+									Collection<Path> libraries) {
```

Comment:
The indentation is really flying away here... I guess it looks better when tabs are shown as 4 spaces and not 8 (GH's default)?

Reply:
Ill just make this 1 line, blame idea for this.